test/scss/assert/_values.scss

```diff
@@ -50,21 +50,20 @@
   }
 
   @include it('Adding floats') {
-    @include assert-equal(0.1 + 0.2, 0.3, $inspect: true);
+    @include assert-equal(0.1 + 0.2, 0.3);
```

Comment:
For colors and math, I'm not sure what we suggest other than not using `$inspect: true`?